src/utils/hitConverter.js

```diff
@@ -8,13 +8,13 @@ const unflatten = require('flat').unflatten;
 
 class HitConverter{
 
-    constructor(customSymbolizers = []){
+    constructor(customSymbolizer = undefined){
```

Comment:
Is defaulting to undefined necessary?

Reply:
I removed it.  I put it there as a hint that it doesn't have to be set.